src/components/common/searchbar.tsx

```diff
@@ -21,21 +23,28 @@ const SearchBar: FC<Props> = ({
 }) => {
   return (
     <div className="block">
-      <CustomInputText
-        type="text"
-        className="w-full min-w-[400px] h-10 !bg-gray-100  p-4 !rounded-full hidden sm:block focus:outline-none placeholder:text-stone-500 placeholder:font-helvetica placeholder:font-medium !shadow-none !border-none"
-        placeholder="Search Order #"
-        value={searchValue}
-        onChange={(e) => setSearchValue(e.target.value)}
-      />
+      <span className="search-input p-input-icon-left hidden sm:block">
+        <CustomImage
+          src={HeaderSearch}
+          alt="Search"
+          className="absolute left-5 bottom-3"
+        />
+        <CustomInputText
+          type="text"
+          className="w-full min-w-[400px] h-[40px] !rounded-full focus:outline-none  !bg-[#30343B] placeholder:text-[#FAF9F6]placeholder:font-helvetica placeholder:font-normal !shadow-none !border-none"
```

Comment:
There is no space in between `placeholder:text-[#FAF9F6]placeholder:font-helvetica` 
Check if style is applied as expected

Reply:
Updated the code